Dockerfile

```diff
@@ -71,6 +71,9 @@ RUN git clone --depth 1 https://github.com/hpc/mpifileutils.git \
 
 FROM mpioperator/openmpi:0.3.0
```

Comment:
While you're here, maybe you could try release 0.4.0.  I see that was created in April.

Reply:
Good call.